spec/spec_helper.rb

```diff
@@ -1,7 +1,12 @@
+require "codeclimate-test-reporter"
+CodeClimate::TestReporter.start
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 
 require 'bundler/setup'
 require 'rspec'
 
+require 'coveralls'
```

Comment:
I fixed rubocop rules + hound rules so these violations will no longer be flagged.